source/common/network/utility.cc

```diff
@@ -373,6 +373,20 @@ absl::uint128 Utility::flipOrder(const absl::uint128& input) {
   return result;
 }
 
+Address::InstanceConstSharedPtr
+Utility::protobufAddressToAddress(const envoy::api::v2::core::Address& proto_address) {
+  switch (proto_address.address_case()) {
+  case envoy::api::v2::core::Address::kSocketAddress:
+    return Network::Utility::parseInternetAddress(proto_address.socket_address().address(),
+                                                  proto_address.socket_address().port_value(),
+                                                  !proto_address.socket_address().ipv4_compat());
+  case envoy::api::v2::core::Address::kPipe:
+    return std::make_shared<Address::PipeInstance>(proto_address.pipe().path());
+  default:
+    throw EnvoyException("Address must be a socket or pipe: " + proto_address.DebugString());
```

Comment:
I think I would do `NOT_REACHED` here. This should never happen w/ schema checks.

Reply:
Done.